docs/install/upgrade.md

```diff
@@ -48,9 +48,10 @@ So, copying `notebook` and `conf` directory should be enough.
 ## Migration Guide
 
 ### Upgrading from Zeppelin 0.6 to 0.7
-
- - From 0.7, we don't use `ZEPPELIN_JAVA_OPTS` as default value of `ZEPPELIN_INTP_JAVA_OPTS` and also the same for `ZEPPELIN_MEM`/`ZEPPELIN_INTP_MEM`. If user want to configure the jvm opts of interpreter process, please set `ZEPPELIN_INTP_JAVA_OPTS` and `ZEPPELIN_INTP_MEM` explicitly. If you don't set `ZEPPELIN_INTP_MEM`, Zeppelin will set it to `-Xms1024m -Xmx1024m -XX:MaxPermSize=512m` by default.
+ - From 0.7, the support on Spark 1.1.x to 1.3.x is deprecated.
+ - Zeppelin embedded Spark won't work anymore. You need to run `./bin/zeppelin-daemon.sh get-spark` or `./bin/zeppelin.sh get-spark` at least one time. Please see [local Spark mode](../interpreter/spark.html#local-spark-mode) for more detailed information.
```

Comment:
As this looks like a substantial change, may be we could use a bit stronger language here - i.e

```
Apache Zeppelin releases do not come with Apache Spark build-in by default any more. In order to be able to run Apache Spark paragraphs, please either run `./bin/zeppelin-daemon.sh get-spark` or point `$SPARK_HOME` to Apache Spark installation. See .... for more details ..
```

What do you think?